### PR TITLE
Document CRL usage

### PR DESCRIPTION
**Pull request by kfaseela**
Document usage for CRL support.

Implementation:

https://github.com/istio/api/pull/2796
https://github.com/istio/istio/pull/45104

### CODE REVIEW

File: content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 1]
```diff
@@ -235,6 +235,14 @@ Follow [these steps](/docs/tasks/traffic-management/egress/egress-gateway-tls-or
 
     The secret **must** be created in the same namespace as the client pod is deployed in, `default` in this case.
 
+    Optionally the credential may include a [certificate revocation list(CRL)]( https://datatracker.ietf.org/doc/html/rfc5280)
```
Comment (ericvn) [inline content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md:238]:
```suggestion
    Optionally the credential may include a [certificate revocation list(CRL)](https://datatracker.ietf.org/doc/html/rfc5280)
```

File: content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 2]
```diff
@@ -522,6 +522,16 @@ You can extend your gateway's definition to support [mutual TLS](https://en.wiki
       --from-file=ca.crt=example_certs1/example.com.crt
     {{< /text >}}
 
+   Optionally the gateway credential may include a [certificate revocation list(CRL)](https://datatracker.ietf.org/doc/html/rfc5280) using the key `ca.crl`. So, the above example would be modified as:
+
+    {{< text bash >}}
+    $ kubectl create -n istio-system secret generic httpbin-credential  \
+      --from-file=tls.key=example_certs1/httpbin.example.com.key \
+      --from-file=tls.crt=example_certs1/httpbin.example.com.crt \
+      --from-file=ca.crt=example_certs1/example.com.crt
```
Comment (ericvn) [inline content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md:531]:
```suggestion
      --from-file=ca.crt=example_certs1/example.com.crt \
```

File: content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md [hunk 3]
```diff
@@ -410,6 +410,14 @@ to hold the configuration of the NGINX server:
...
     In this example. a single generic Secret with keys `tls.key`, `tls.crt`, and `ca.crt` is used.
 
+    Optionally the gateway credential may include a [certificate revocation list(CRL)](https://datatracker.ietf.org/doc/html/rfc5280)
+    using the key `ca.crl`. So, the above example would be modified as:
+
+    {{< text bash >}}
+    $ kubectl create secret -n istio-system generic client-credential --from-file=tls.key=client.example.com.key \
+      --from-file=tls.crt=client.example.com.crt --from-file=ca.crt=example.com.crt --from-file=ca.crl=/some/path/to/your-crl.pem
+    {{< /text >}}
+
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md:420]: How about making this a tip and putting into a boilerplate?

The tip could be something like this:

```
{{< tip >}}
Optionally, the gateway credential may include a [certificate revocation list(CRL)](https://datatracker.ietf.org/doc/html/rfc5280) using the key `ca.crl`.
If so, add another argument to the above example to provide your CRL: `--from-file=ca.crl=/some/path/to/your-crl.pem` 
{{< /tip >}}
```

Comment (kfaseela) [reply content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md:420]: I think this makes better sense.

Comment (kfaseela) [reply content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md]: @frankbu @ericvn changed according to your suggestion. Please have another look

File: content/en/boilerplates/crl-tip.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+---
+{{< tip >}}
+Optionally the credential may include a [certificate revocation list (CRL)](https://datatracker.ietf.org/doc/html/rfc5280)
```
Comment (frankbu) [inline content/en/boilerplates/crl-tip.md:4]: nit
```suggestion
Optionally, the credential may include a [certificate revocation list (CRL)](https://datatracker.ietf.org/doc/html/rfc5280)
```

**Review by frankbu — COMMENTED** [review]
One small nit, but otherwise LGTM

**Review by ericvn — APPROVED** [review]

**Pull request merged**